packages/fxa-auth-server/config/dev.json

```diff
@@ -427,7 +427,7 @@
     "secondInterval": "2s"
   },
   "mjml": {
-    "enabledEmailAddress": "@testuser.com$",
+    "enabledEmailAddress": "@mozilla.com$",
```

Comment:
Replaced this value back to what Vijay has set in this [PR](http://localhost:3030/connect_another_device?utm_medium=email&utm_campaign=fx-cad-reminder-first&utm_content=fx-connect-device)